src/main/java/core/basesyntax/strategy/ReturnOperation.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public class ReturnOperation implements OperationHandler {
+    private final Storage storage;
+
+    public ReturnOperation(Storage storage) {
+        this.storage = storage;
+    }

```

Comment:
According to the encapsulation principle, the `storage` field should be private to hide the inner structure of the class and to prevent unauthorized access and modification from outside of the class.